src/__test__/components/plots-and-tables/markerHeatmap.test.jsx

```diff
@@ -182,14 +182,14 @@ let loadedProcessingConfigSpy;
 const renderHeatmapPage = async (newStore) => {
   store = createStore(rootReducer, _.cloneDeep(newStore), applyMiddleware(thunk));
 
-  rtl.render(
+  render(
     <Provider store={store}>
       <MarkerHeatmap
         experimentId={experimentId}
       />
     </Provider>,
   );
-  await rtl.waitFor(() => expect(loadConfigSpy).toHaveBeenCalledTimes(1));
+  await waitFor(() => expect(loadConfigSpy).toHaveBeenCalledTimes(1));
```

Comment:
what is the purposes of using `expect` outside actual test cases? Unless there is a good reason I find it's just confusing, if possible add it inside the test where it belongs